### ISSUE TITLE
for Windows

### ISSUE THREAD

**Issue by Clown1206** [ray-project/ray#22271]
Hello, I have just come into contact with Ray, can it run on Windows10 ?

**Comment by pcmoritz** [comment]
Yes, you can just run `pip install ray` on windows (or `pip install "ray[default]"`)

See also https://docs.ray.io/en/latest/installation.html
